app/ashlar/models.py

```diff
@@ -58,7 +57,7 @@ class Record(AshlarModel):
     label = models.CharField(max_length=50)  # Human-readable
     slug = models.CharField(max_length=50)
 
-    geom = models.PointField(srid=settings.ASHLAR_SRID)
+    geom = models.PointField(srid=settings.ASHLAR['SRID'])
```

Comment:
Not something to be fixed now, but it occurs to me that this doesn't really play so well with migrations, since the migrations will just copy whatever is in the settings file at the time the migration is generated.